test/optimizer/stats_storage_test.cpp

```diff
@@ -0,0 +1,73 @@
+#include <utility>
+
+#include "gtest/gtest.h"
+#include "optimizer/statistics/stats_storage.h"
+
+#include "util/test_harness.h"
+
+namespace terrier::optimizer {
+
+// NOLINTNEXTLINE
+TEST(StatsStorageTests, GetTableStatsTest) {
+  auto column_stats_obj_1 = ColumnStats(catalog::db_oid_t(1), catalog::table_oid_t(1), catalog::col_oid_t(1), 5, 4, 0.2,
```

Comment:
Now you are really going to want to refactor this to only create these objects once. There is too much repeated code.